packages/frames.js/src/express/index.node.ts

```diff
@@ -0,0 +1,31 @@
+import { webcrypto } from "node:crypto";
+import { createFramesFactory } from "./factory";
+
+export { Button, type types } from "./factory";
+
+/**
```

Comment:
@stephancill you will see multiple comments like this one. The issue is that intellisense is not able to show you documentation from the type so I had to annotate the export explicitly.